remove budget, requirements and transaction_milestones extensions

duncandewhurst opened the pull request:
Closes #233, closes #249

* Remove extensions from `extensions_versions.json`
* Run `build_profile.py`
* Remove extension codelists from `codelists.md`
* Remove extension fields from `full.json`
* Generate `full_record_package.json` using OCDS Kit
* Update changelog

@jpmckinney was anything else required to close #233?

duncandewhurst committed:
remove budget, requirements and transaction_milestones extensions

duncandewhurst committed:
Update changelog

duncandewhurst committed:
add changes from ppp extension

jpmckinney commented on docs/reference/changelog.md:
```diff
@@ -9,11 +9,11 @@
+  * [#245](https://github.com/open-contracting-extensions/public-private-partnerships/pull/245) Remove the preQualification extension and update framework reference accordingly.
+  * [#243](https://github.com/open-contracting-extensions/public-private-partnerships/pull/243) Restore `buyer` and `awards/supplier` fields and associated codes. Remove `publicAuthority` and `awards/preferredBidders` fields and associated codes. Update the framework reference accordingly.
+  * [#244](https://github.com/open-contracting-extensions/public-private-partnerships/pull/244) Restore 'tender' code and remove 'ppp' code from `initiationType.csv`.
+* [#260](https://github.com/open-contracting-extensions/public-private-partnerships/pull/260) Remove extensions that were not referenced in the framework guidance from the profile (budget, requirements and transaction milestones).
```
```suggestion
* [#260](https://github.com/open-contracting-extensions/public-private-partnerships/pull/260) Remove extensions that were not referenced in the framework guidance from the profile (budget breakdown, process-level title and description, requirements and transaction related milestones).
```

jpmckinney replied:
@duncandewhurst Just this comment left to resolve.

jpmckinney reviewed (commented):
From #233, I think we need to add these to `codelists.md`:

* [x] geometryType.csv (location)
* [x] locationGazetteers.csv (location)
* [x] chargePaidBy.csv (charges, tariffs)
* [x] financeType.csv (finance)
* [x] metricID.csv (ppp)

jpmckinney committed:
Merge branch '1.0-dev' into 233-remove-extensions

duncandewhurst committed:
add codelists to codelists.md

add geometryType.csv (location), locationGazetteers.csv (location), chargePaidBy.csv (charges, tariffs), financeType.csv (finance), metricID.csv (ppp)

duncandewhurst committed:
Merge branch '233-remove-extensions' of github.com:open-contracting-extensions/public-private-partnerships into 233-remove-extensions

jpmckinney commented on docs/reference/codelists.md:
````diff
@@ -75,35 +75,35 @@ For additional guidance on using this codelist, refer to the [Requirements exten
 ```
 
-For additional guidance on using this codelist, refer to the [Requirements extension documentation](https://extensions.open-contracting.org/en/extensions/requirements/master/).
+For additional guidance on using this codelist, refer to the [Financing extension documentation](https://extensions.open-contracting.org/en/extensions/finance/master/).
````
```suggestion
For additional guidance on using this codelist, refer to the [Finance extension documentation](https://extensions.open-contracting.org/en/extensions/finance/master/).
```

duncandewhurst committed:
Update docs/reference/codelists.md

Co-authored-by: James McKinney <26463+jpmckinney@users.noreply.github.com>

jpmckinney committed:
codelists: Change releaseTag to link

duncandewhurst committed:
Update docs/reference/changelog.md

Co-authored-by: James McKinney <26463+jpmckinney@users.noreply.github.com>
